test/tests/userobjects/moab_skinner/expanding-cube/all_bins.i

```diff
@@ -0,0 +1,139 @@
+[Mesh]
+ [cube]
+    type = FileMeshGenerator
+    file = '../../../neutronics/meshes/tet_cube.e'
+ []
+ [id1]
```

Comment:
Is there a reason why the subdomains are being modified like this? Is this essential for the test? If not, let's remove it because it is extraneous.